src/components/AuthWrapper/AuthWrapper.tsx

```diff
@@ -42,6 +43,7 @@ export const AuthWrapper: FC<Props> = props => {
                 <ConfirmSignUp authProps={props} path="/confirm/:code" />
 
                 <ApplicationPage path="/application/:id" />
+                <QuestionPage path="/question/:id" />
```

Comment:
Don't put this in this component. This is for wrapping the authentication components.

Reply:
Where would I put this? I tried initially in App.tsx, following the pattern on the other projects, however the page will load and then always be redirected back to "/"

Reply:
Fixed